py/picca/fitter2/chi2.py

```diff
@@ -31,6 +31,8 @@ def __init__(self,dic_init):
             else:
                 self.seedfast_mc = 0
             self.nfast_mc = dic_init['fast mc']['niterations']
+            if 'covscaling' in dic_init['fast mc']:
+                self.scalefast_mc = dic_init['fast mc']['covscaling']
```

Comment:
set to 1 if not given

Reply:
I considered it but found it unnecessary to define and store it if it's unity (we don't store an empty list of priors when there are none). I can change it though if you want.

Reply:
it is simply that with setting it to 1 by default you don't have to use if hasattr(self,"scalefast_mc").
As you want.